pizza-time/src/components/events/SingleEvent/info.js

```diff
@@ -651,20 +651,7 @@ class Info extends React.Component {
               </div>
 
               <div className="invite-switch">
-                <div className="organizer-info">
-                  {this.state.organizerAvatar.length > 0 ? (
-                    <>
-                      <div className="event-info-label">Organizer: </div>
```

Comment:
please keep my code. your version is older than the development you are merging to